test/games/strategy/engine/data/annotations/ValidateAttachmentsTest.java

```diff
@@ -177,6 +177,9 @@ private static String findAttachmentsAndValidate(final File file) {
       final int index = fileName.indexOf(classesRoot) + classesRoot.length();
       String className = fileName.substring(index);
       className = className.replace(File.separator, ".");
+      if( !className.endsWith(".class")) {
+        return "";
+      }
```

Comment:
I didn't address your question as directly as I could have. All classes get compiled into .class, but some non class files can make it into the search path of this unit test if you run build steps in certain sequences.